packages/scandipwa/src/store/MyAccount/MyAccount.dispatcher.js

```diff
@@ -101,6 +101,9 @@ export class MyAccountDispatcher {
         dispatch(updateCustomerSignInStatus(false));
         dispatch(updateCustomerDetails({}));
 
+        const mutation = MyAccountQuery.getRevokeAccountToken();
```

Comment:
We can not wait for answer since we don't depend on it. It will just delete token from DB and even if error appears, nothing should happen on FE.

Reply:
Then maybe you can handle when it's done by using `fetchMutation().finally(() => {})` ?